noxfile.py

```diff
@@ -43,7 +47,7 @@ def coverage(session):
 
 @nox.session(python=PYTHON_VERSIONS)
 def lint(session):
-    deps(session)
+    deps(session, local_install=True)
```

Comment:
we are not actually linting the generated code anyway. why do we need it installed with -e?

Reply:
As mentioned above, it doesn't really matter, it just made more sense to me to use the local code since it is the one we are linting. That being said, you bring up a good point: if in the future there is some kind of code generated during `pip install` that we want to validate, then it would become important.

Reply:
See if if you can eliminate the local_mode.

Reply:
Let's continue this thread of discussion in https://github.com/omry/omegaconf/pull/321#discussion_r513670942